app/js/components/orderEntry/PastOrder.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+export const PastOrder = props => (
+  <tr>
+    <td>
+      {format(props.auditInfo.dateCreated, 'DD/MM/YYYY | HH:MM:SS')}
+    </td>
+    <td>{props.drug.display}: {props.dose} {props.doseUnits.display}, {props.frequency.display}, {props.route.display} for {props.duration} {props.durationUnits.display} total (Dispense: {props.quantity} {props.quantityUnits.display})</td>
```

Comment:
airbnb requires not more than 80 characters on a single line, please can you break this down

Reply:
Worked on